apps/stable_diffusion/src/utils/sd_annotation.py

```diff
@@ -0,0 +1,191 @@
+import os
+from shark.model_annotation import model_annotation, create_context
+from shark.iree_utils._common import iree_target_map, run_cmd
+from shark.shark_downloader import (
+    download_model,
+    download_public_file,
+    WORKDIR,
+)
+from shark.parser import shark_args
+from .stable_args import args
+
+
+device = (
+    args.device if "://" not in args.device else args.device.split("://")[0]
+)
```

Comment:
avoid such global variables.

Reply:
agree here and elsewhere with what gaurav said. We want to move away from global variables and dependence on args outside of the script that is called from the command line